### PR TITLE
Present UI

### PR DESCRIPTION
**Pull request by fadezak100**
- Added props to the navbar to change the 'sign in' text to 'logout' when user is logged in
- Created a card component for presentations that takes two props, `newId`: to link the toggle button with its card.`type`: to specify whether the presentation is uploading or already have been uploaded.
- Created a component for adding a new presentation.
- Installed `react-router-dom`
- deleted `app.tsx` and configured the router in the `index.tsx` file.

PS: I saw a router file in the front-end file structure, but I couldn't figure out how to use it properly.

### CODE REVIEW

File: client/src/components/Navbar/index.tsx [hunk 1]
```diff
@@ -1,19 +1,28 @@
 import logo from '../../assets/Logo/About.png'
 import './styles.css'
 
-const Navbar = () => {
+interface Props {
```
Comment (AhmedSafi97) [inline client/src/components/Navbar/index.tsx:4]: naming

File: client/src/index.tsx [hunk 2]
```diff
@@ -1,11 +1,26 @@
 import React from 'react'
 import ReactDOM from 'react-dom/client'
-import App from './App'
 import './index.css'
+import Landing from './pages/Landing/Landing'
+import Presentations from './pages/Presentation'
+
+import { createBrowserRouter, RouterProvider } from 'react-router-dom'
+
+const router = createBrowserRouter([
```
Comment (AhmedSafi97) [inline client/src/index.tsx:9]: restore App and move routing logic to the app

**Review by AhmedSafi97 — CHANGES_REQUESTED** [review]

File: client/src/index.tsx (hunk 2 above)
Comment (moustf) [reply client/src/index.tsx:9]: Agreed!

File: client/src/components/PresentationCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
+import { ReactComponent as PresentationIcon } from '../../assets/PresentationIcons/presentationIcon.svg'
+import { ReactComponent as LivePresentation } from '../../assets/PresentationIcons/livePresentation.svg'
+import { ReactComponent as DeletePresentation } from '../../assets/PresentationIcons/deletePresentation.svg'
+
+interface Props {
+  newId: string
+  type: string
+}
```
Comment (moustf) [inline client/src/components/PresentationCard/index.tsx:8]: As Safi said, naming!

File: client/src/components/Navbar/index.tsx (hunk 1 above)
Comment (fadezak100) [reply client/src/components/Navbar/index.tsx]: Done

File: client/src/components/PresentationCard/index.tsx (hunk 3 above)
Comment (fadezak100) [reply client/src/components/PresentationCard/index.tsx:8]: thanks ya expensive

**Review by AhmedSafi97 — APPROVED** [review]

**Pull request merged**